src/app_engine/compute_page.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+"""Compute page for handling tasks related to compute engine."""
+
+import logging
+
+import webapp2
+
+import apiauth
+from google.appengine.api import app_identity
+from google.appengine.api import taskqueue
+
+
+# Compute page constants
+ACTION_START = 'start'
+ACTION_STATUS = 'status'
+
+# Constants for the Compute Engine API
+COMPUTE_STATUS = 'status'
+COMPUTE_STATUS_TERMINATED = 'TERMINATED'
+COMPUTE_STATUS_RUNNING = 'RUNNING'
+
+# Seconds between API retries.
+TASK_RETRY_S = 3
+
+COMPUTE_API_URL = 'https://www.googleapis.com/auth/compute'
+
+
+class ComputePage(webapp2.RequestHandler):
+  """Page to handle requests against GCE."""
+
+  compute_service = None
+
+  def __init__(self, request, response):
+    self.initialize(request, response)
```

Comment:
yeah i would normally do that but I read this: https://webapp-improved.appspot.com/guide/handlers.html#overriding-init

apparently that's the way they prescribe but yeah, I started out with calling the superclass function. maybe they think this is more explicit looking and less magic?